user/design/ms-iis-pack.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: user-doc
+title: Microsoft IIS Pack
+---
+
+The _iis_ [pack](./packs.html) is available as _Internet Information Services(IIS)_ in the _Web Application_ section
+and provides the user with the ability to use [Microsoft IIS](https://www.iis.net/) as a platform in their assembly.
+
+
+There are several main components of the IIS Pack and each are addressed below.
+
+### compute Component
+Used to define the Windows VM compute size.
+Default compute size: M-WIN (defined in your local instance of OneOps).
+Please refer to the [compute component](./compute-component.html) for additional info.
+
+### dotnetframework Component
+Used to install different .net frameworks on windows compute/node.
+This component uses Chocolatey to install .net frameworks.
+
+Attribute | Description | Notes
+--------- | ----------- | ---------------------
+Package url | Url of the chocolatey repository | Default Package URL: https://chocolatey.org/api/v2/.  Add the url of the chocolatey package source, this will be overridden if mirror cloud service has been defined.  In the mirror cloud service the key to used be is "chocorepo".
+Net Framework version | Add the .Net framework version details | Format: .Net framework version = chocolatey package name.  (Ex.  .Net 3.5 = dotnet3.5) Default Net Framework versions: .Net 3.5 = dotnet3.5,   .Net 4.5.2 = dotnet4.5.2
+
+### iis-framework Component
+Used to define the website, application pool and configure different attributes in IIS.
+
+##### IIS Website section
```

Comment:
Same logic here . if its part of the iss-framework component it should use ## for the component and ### for the nested folders. it could also be a standalone doc for that component and you just link to it.